commons/src/main/kotlin/org/fossify/commons/extensions/Context.kt

```diff
@@ -591,6 +591,41 @@ fun Context.formatSecondsToTimeString(totalSeconds: Int): String {
     return result
 }
 
+fun Context.formatMinutesToShortTimeString(totalMinutes: Int) = formatSecondsToShortTimeString(totalMinutes * 60)
+
+fun Context.formatSecondsToShortTimeString(totalSeconds: Int): String {
+    val days = totalSeconds / DAY_SECONDS
+    val hours = (totalSeconds % DAY_SECONDS) / HOUR_SECONDS
+    val minutes = (totalSeconds % HOUR_SECONDS) / MINUTE_SECONDS
+    val seconds = totalSeconds % MINUTE_SECONDS
+    val timesString = StringBuilder()
+    if (days > 0) {
+        val daysString = String.format(resources.getString(R.string.days_short), days)
+        timesString.append("$daysString, ")
```

Comment:
I don't think we need the comma here.

`1h, 23m, 15s` doesn't flow as naturally as `1h 23m 15s`.

```suggestion
        timesString.append("$daysString ")
```